src/Arduino/handcontroll_arduino.ino

```diff
@@ -8,11 +9,15 @@
 // ServoController Class
 
 class ServoControl {
-  
+  private:
+     Adafruit_PWMServoDriver Servos;
   public:
 
-    ServoControl(){
-        // initilize Servos
+    ServoControl(){   // initilize Servocontrol
+        Servos = Adafruit_PWMServoDriver();
+        Servos.begin();
+        Servos.setPWMFreq(60);
+        delay(10);
     };
 
     virtual ~ServoControl(){
```

Comment:
Kann das ServoBoard irgendwie wieder geschlossen werden.
Quasi `Servos.close()` oder sowas?

Reply:
Nein, hab nochmal in die Doku geschaut, leider nicht